src/views/empty-state.component.js

```diff
@@ -11,6 +11,14 @@
  *   <li>.icon   - (string) class for main icon. Ex. 'pficon pficon-add-circle-o'
  *   <li>.title  - (string) Text for the main title
  *   <li>.info  - (string) Text for the main informational paragraph
+ *   <li>.helpLink - (object) Contains url specific properties and actions
+ *   <ul style='list-style-type: none'>
+ *     <li>.label - (string) Text label which appears before the
+ *     <li>.urlLabel - (string) Text for the clickable portion of the link
+ *     <li>.url - (string) Text for url path
+ *     <li>.urlAction - (function) Function to invoke a url action when a callback method is specified.
```

Comment:
This is optional, right?

Reply:
Yes. Will make sure to note that

Reply:
looks like helpLink and it's properties are under the "config Optional configuration object" for the config param. Would I still need to note that it's optional?

Reply:
I would still mark them as optional for that object.

Reply:
Okay will do.